docs/code_coverage.md

```diff
@@ -45,12 +47,13 @@ python infra/helper.py profile $project_name
 ```
 
 If you want to generate code coverage report using the corpus you have locally,
-copy the corpus into `build/$project_name/$fuzz_target/` directories for each
-fuzz target, then run:
+copy the corpus into `build/corpus/$project_name/$fuzz_target/` directories for
```

Comment:
This is not good solution, why not provide a command line parameter that allows any local corpus path and then we mount it using -v. Can be done in next cl.

Reply:
Sure, agreed. I've just tried to simplify https://github.com/google/oss-fuzz/pull/1494/files which was already quite big.

Reply:
Uploaded https://github.com/google/oss-fuzz/pull/1535 as per out discussion.